include/irr/asset/ICPUMesh.h

```diff
@@ -16,8 +16,7 @@ class ICPUMesh : public IMesh<ICPUMeshBuffer>, public BlobSerializable, public I
 	public:
 		//! These are not absolute constants, just the most common situation, there may be setups of assets/resources with completely different relationships.
 		_IRR_STATIC_INLINE_CONSTEXPR uint32_t MESHBUFFER_HIERARCHYLEVELS_BELOW = 1u; // mesh->meshbuffer->texture
-		// TODO: after shader pipeline change to 5, mesh->meshbuffer->descriptorset->imageview
-		_IRR_STATIC_INLINE_CONSTEXPR uint32_t IMAGEVIEW_HIERARCHYLEVELS_BELOW = 2u; // mesh->meshbuffer->texture
+		_IRR_STATIC_INLINE_CONSTEXPR uint32_t IMAGEVIEW_HIERARCHYLEVELS_BELOW = 5u; // mesh->meshbuffer->texture
```

Comment:
actually mesh->meshbuffer->pipeline->descriptorset->imageview would make it 4,
5 would be for `IMAGE_HIERARCHYLEVELS_BELOW`

Reply:
@Crisspl TODO in next PR